backend/src/main/java/pl/jcommerce/ewypowiedzenia/api/DismissalNoticeController.java

```diff
@@ -5,25 +5,29 @@
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 import pl.jcommerce.ewypowiedzenia.application.DismissalNoticeService;
 import pl.jcommerce.ewypowiedzenia.infrastructure.DismissalNoticeDto;
 import pl.jcommerce.ewypowiedzenia.infrastructure.FileDto;
 
 @RestController
+@CrossOrigin(origins = "*")
```

Comment:
Hmmm... Why do we need this ? Front-end part works on different port and domains do not match ?

Reply:
Exactly - different ports.